Userland/mount.cpp

```diff
@@ -179,7 +199,9 @@ int main(int argc, char** argv)
             fs_type = "ext2";
         int flags = options ? parse_options(options) : 0;
 
-        if (mount(source, mountpoint, fs_type, flags) < 0) {
+        int fd = get_source_fd(source);
```

Comment:
Missing an error check here.

Reply:
There's not supposed to be an error check: if we get `-1`, we pass just that to the kernel.

Reply:
Okydoky!

Reply:
Maybe we should print an error message inside `get_source_fd()` if we fail to open the file though, just for diagnostics.